components/clarinet-cli/src/frontend/cli.rs

```diff
@@ -94,11 +95,31 @@ enum Command {
     /// Get Clarity autocompletion and inline errors from your code editor (VSCode, vim, emacs, etc)
     #[clap(name = "lsp", bin_name = "lsp")]
     LSP,
+    /// Format clarity code files
+    #[clap(name = "format", aliases = &["fmt"], bin_name = "format")]
+    Formatter(Formatter),
     /// Step by step debugging and breakpoints from your code editor (VSCode, vim, emacs, etc)
     #[clap(name = "dap", bin_name = "dap")]
     DAP,
 }
 
+#[derive(Parser, PartialEq, Clone, Debug)]
+struct Formatter {
+    /// Path to clarity files (defaults to ./contracts)
+    #[clap(long = "path", short = 'p')]
+    pub code_path: Option<String>,
```

Comment:
By default, the formatter should be based on the manifest.
Load the manifest like other comamnds do (looking for `./Clarinet.toml`). And an option should allow to specify the manifest path